cli/lsp/analysis.rs

```diff
@@ -620,9 +621,13 @@ fn try_reverse_map_package_json_exports(
 pub fn fix_ts_import_changes(
   changes: &[tsc::FileTextChanges],
   language_server: &language_server::Inner,
+  token: &CancellationToken,
 ) -> Result<Vec<tsc::FileTextChanges>, AnyError> {
   let mut r = Vec::new();
   for change in changes {
+    if token.is_cancelled() {
+      return Err(anyhow!("request cancelled"));
```

Comment:
How do you decide if we should return an error vs and empty vector?

Reply:
At some point I decided they should all error, to reduce the chance of later code dealing with nonsense data.